app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Utils/LocationAutocomplete.java

```diff
@@ -0,0 +1,48 @@
+package ithaca_transit.android.cornellappdev.com.ithaca_transit.Utils;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.arlib.floatingsearchview.suggestions.model.SearchSuggestion;
+
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.Models.Place;
+
+public class LocationAutocomplete implements SearchSuggestion {
+
+    public static final Parcelable.Creator CREATOR = new Parcelable.Creator() {
+        public LocationAutocomplete createFromParcel(Parcel in) {
+            return new LocationAutocomplete(in);
+        }
+
+        public LocationAutocomplete[] newArray(int size) {
+            return new LocationAutocomplete[size];
+        }
+    };
+
+    Place data;
+    String desc;
```

Comment:
should these be private?

Reply:
Probably, setting them as such doesn't seem to affect the autocomplete results so I've gone and done so.